bitcoin/src/blockdata/opcodes.rs

```diff
@@ -67,7 +67,7 @@ macro_rules! all_opcodes {
         pub static OP_NOP3: Opcode = OP_CSV;
 
         impl fmt::Display for Opcode {
```

Comment:
`core` should be added here and also below for `fmt::Formatter`.

Reply:
Fixed